strawberry/types/fields/resolver.py

```diff
@@ -294,16 +294,24 @@ def copy_with(
         if self.type:
             if isinstance(self.type, StrawberryType):
                 type_override = self.type.copy_with(type_var_map)
-            else:
-                type_override = self.type._type_definition.copy_with(  # type: ignore
+            elif hasattr(self.type, "_type_definition"):
+                type_override = self.type._type_definition.copy_with(
```

Comment:
do you think we can add coverage for this line? :)

Reply:
It's not covered now, which make me wonder if it's necessary.